modules/statics/src/coins.ts

```diff
@@ -1120,4 +1124,22 @@ export const coins = CoinMap.fromCoins([
   teosToken('teos:IQ', 'Everipedia', 3, 'testtoken112', UnderlyingAsset.IQ, AccountCoin.DEFAULT_FEATURES, '', 'IQ'),
   teosToken('teos:BOX', 'Box', 6, 'kvszn1xyz1bu', UnderlyingAsset.EOS_BOX, AccountCoin.DEFAULT_FEATURES, '', 'BOX'),
   teosToken('teos:USDT', 'Tether', 4, 'lionteste212', UnderlyingAsset.USDT, AccountCoin.DEFAULT_FEATURES, '', 'USDT'),
+  avaxErc20('QI', 'BenQi', 18, '0x8729438eb15e2c8b576fcc6aecda6a148776c0f5', UnderlyingAsset['avaxc:QI']),
+  avaxErc20('XAVA', 'Avalaunch', 18, '0xd1c3f94de7e5b45fa4edbba472491a9f4b166fc4', UnderlyingAsset['avaxc:XAVA']),
+  avaxErc20('KLO', 'Kalao', 18, '0xb27c8941a7df8958a1778c0259f76d1f8b711c35', UnderlyingAsset['avaxc:KLO']),
+  avaxErc20('JOE', 'Trader Joe', 18, '0x6e84a6216ea6dacc71ee8e6b0a5b7322eebc0fdd', UnderlyingAsset['avaxc:JOE']),
+  avaxErc20('PNG', 'Pangolin', 18, '0x60781c2586d68229fde47564546784ab3faca982', UnderlyingAsset['avaxc:PNG']),
+  avaxErc20('USDT', 'Tether', 6, '0xc7198437980c041c805a1edcba50c1ce5db95118', UnderlyingAsset.USDT),
+  tavaxErc20('tQI', 'Test BenQi', 18, '0x8729438eb15e2c8b576fcc6aecda6a148776c0f5', UnderlyingAsset['avaxc:QI']),
+  tavaxErc20(
+    'tXAVA',
+    'Test Avalaunch',
+    18,
+    '0xd1c3f94de7e5b45fa4edbba472491a9f4b166fc4',
+    UnderlyingAsset['avaxc:XAVA']
+  ),
+  tavaxErc20('tKLO', 'Test Kalao', 18, '0xb27c8941a7df8958a1778c0259f76d1f8b711c35', UnderlyingAsset['avaxc:KLO']),
+  tavaxErc20('tJOE', 'Test Trader Joe', 18, '0x6e84a6216ea6dacc71ee8e6b0a5b7322eebc0fdd', UnderlyingAsset['avaxc:JOE']),
+  tavaxErc20('tPNG', 'Test Pangolin', 18, '0x60781c2586d68229fde47564546784ab3faca982', UnderlyingAsset['avaxc:PNG']),
+  tavaxErc20('tUSDT', 'Test Tether', 6, '0xc7198437980c041c805a1edcba50c1ce5db95118', UnderlyingAsset.USDT),
```

Comment:
should USDC be here?

Reply:
there isn't information in the PRD about usdc